kernel/build-kernel.sh

```diff
@@ -164,11 +178,21 @@ get_kernel_frag_path() {
 	# handle specific cases, then add the path definition and search/list/cat
 	# here.
 	local all_configs="${common_configs} ${arch_configs}"
+	if [[ ${experimental_kernel} == "true" ]]; then
+		all_configs="${all_configs} ${experimental_configs}"
+	fi
 
+	temp_file=$(mktemp)
 	info "Constructing config from fragments: ${config_path}"
-	local results=$(export KCONFIG_CONFIG=${config_path}; \
-					export ARCH=${arch_target}; \
-					cd ${kernel_path}; ${cmdpath} -r -n ${all_configs})
+
+	export KCONFIG_CONFIG=${config_path}
+	export ARCH=${arch_target}
+	cd ${kernel_path}
+	${cmdpath} -r -n ${all_configs} &> ${temp_file}
```

Comment:
(Channelling @grahamwhaley)... Could we do this without the temp file? Something like:

```sh
local result=$( ${cmdpath} -r -n ${all_configs} )

grep "${not_in_string}" <<< "$result"
```

Reply:
I suffered greatly trying to make it work. Did what was in my bash toolbox. If you can make it also filter out results from whitelist I’m happy to do this. 

Suggestions?

Reply:
I think you should be able to do:

```sh
local results

results=$( ${cmdpath} -r -n ${all_configs} )
results=$(grep "${not_in_string}" <<< "$results")
results=$(grep -v -f ${default_config_whitelist} <<< "$results")
```

Reply:
Addressed - Thanks James.  This works for me.